src/AasxServerDB/Converter.cs

```diff
@@ -85,40 +85,35 @@ static private void LoadSME(Submodel submodel, ISubmodelElement? sme, string? SM
             foreach (var smel in smeLevel)
             {
                 ISubmodelElement? nextSME = null;
+                var value = smel.getValue();
                 switch (smel.SMEType)
                 {
                     case "Prop":
-                        nextSME = new Property(DataTypeDefXsd.String, idShort: smel.IdShort, value: smel.getValue());
+                        nextSME = new Property(DataTypeDefXsd.String, value: value.First()[0]);

```

Comment:
Same here. If you use .First and that returns null we get an exception

Reply:
The method is designed to always return at least the default value.

Reply:
yes. which is an empty list. And when it is empty and you want to access it first element with `[0]` than there is no element, so you get an exception

Reply:
![image](https://github.com/eclipse-aaspe/server/assets/168823574/9f6ab08d-b503-4292-a83e-a2d4bdfea312)

Reply:
I wanted to avoid always having to check for the first Element because of multiple code and confusing.

Reply:
Can you rebase with the current main. When I try this I get the following issue:

![image](https://github.com/eclipse-aaspe/server/assets/10234796/9381db3a-6a55-4b57-9ee7-8995c9b73b70)